contrib/win32/install/openssh.wixproj

```diff
@@ -14,6 +14,7 @@
             ProductVersion=$(ProductVersion);
         </DefineConstants>
         <DefineSolutionProperties>false</DefineSolutionProperties>
+        <!-- This is sufficient for x86 and x64 but gets overwritten below for ARM64 -->
```

Comment:
I don't understand this comment.  Is this Ok for Arm64 or does something need to be done to fix the 'overwrite'?